src/Rings/mpoly-affine-algebras.jl

```diff
@@ -19,20 +19,54 @@ julia> dim(A)
 1
 ```
 """
-function dim(A::MPolyQuoRing)
-  return dim(modulus(A))
+dim(A::Union{MPolyQuoRing, zzModRing, ZZModRing}) = krull_dim(A)
```

Comment:
```suggestion
dim(A::MPolyQuoRing) = krull_dim(A)
```
We really should not force `dim` for `ZZModRing` and `zzModRing` to be the Krull dimension. This is also type piracy.

Reply:
@jankoboehm and myself agree with this.